marin/execution/executor.py

```diff
@@ -194,6 +210,14 @@ def hardcoded(path: str) -> "InputName":
         """
         return InputName(None, name=path)
 
+    def skip_parent(self, should_skip: bool = True) -> "InputName":
```

Comment:
`should_skip` isn't intuitive to me - maybe `dont_wait_for_finish` or `impatient`?

Reply:
i switched to nonblocking()